azurerm/resource_arm_application_gateway.go

```diff
@@ -323,6 +323,34 @@ func resourceArmApplicationGateway() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+
+						"custom_error_configurations": {
```

Comment:
I see. It is in plural because I followed the exact name in the Azure SDK, but sure I can rename it.
`CustomErrorConfigurations *[]ApplicationGatewayCustomError json:"customErrorConfigurations,omitempty"`